components/brave_new_tab_ui/components/default/braveNews/FeedV2.tsx

```diff
@@ -57,7 +57,7 @@ export default function FeedV2() {
 
   // Note: Whenever the feed is updated, if we're viewing the feed, scroll to
   // the top.
-  React.useEffect(() => {
+  React.useLayoutEffect(() => {
```

Comment:
I think this documentation from https://react.dev/reference/react/useEffect exactly matches this situation 👍🏼 

```
If your Effect wasn’t caused by an interaction (like a click), React will generally let the browser paint the
updated screen first before running your Effect. If your Effect is doing something visual (for example,
positioning a tooltip), and the delay is noticeable (for example, it flickers), replace useEffect with
useLayoutEffect.
```